### PR TITLE
ELPP-3522 Remove CloudFront from InvalidateCdn activity

### PR DESCRIPTION
**Pull request by giorgiosironi**
These CDNs are not used anymore in favor of the Fastly ones, so invalidation can be faster.

Incidentally I also found some settings that are unused (as far as I can tell from looking at the code) and that relate to CloudFront - better to remove them altogether to avoid confusion.

### CODE REVIEW

File: activity/activity_InvalidateCdn.py [hunk 1]
```diff
@@ -30,38 +29,10 @@ def do_activity(self, data):
             return activity.activity.ACTIVITY_PERMANENT_FAILURE
 
         try:
-
-            self.emit_monitor_event(self.settings, article_id, version, run,
```
Comment (gnott) [inline activity/activity_InvalidateCdn.py]: At the start of the activity, it should still have a ``self.emit_monitor_event()`` with a ``"start"`` message to the dashboard, which is later closed with the ``"end"`` message you have on line 36.

File: settings-example.py [hunk 2]
```diff
@@ -71,15 +71,8 @@ class exp():
...
     ses_admin_email = "admin@example.com"
 
-    # CDN bucket settings
-    cdn_bucket = 'elife-cdn-dev'
-    cdn_distribution_id = u'E1HPZ2QWOYE9NX'
-    cdn_domain_name = 'dhkzd83nokruv.cloudfront.net'
-
     # Lens bucket settings
     lens_bucket = 'elife-lens-dev'
-    lens_distribution_id = u'E30WWCB2DNEOKI'
```
Comment (gnott) [inline settings-example.py]: It's probably ok to drop the Lens CDN settings, which is a separate CDN, but I don't think we implemented invalidating the content. We do invalidate the XML file Lens loads (which is stored in the primary CDN) as well as the figures, which I think results in an updated Lens page. Lens invalidations would only matter if we changed the Lens template, or the Lens source files (``.js`` and ``.css`` files).

Comment (giorgiosironi) [reply settings-example.py]: Invalidation is only an optimization as the caching of the files will still expire after the default TTL. The invalidation implemented here was for the production team wanting to get fast feedback from the preview site about a new ingestion.

File: activity/activity_InvalidateCdn.py (hunk 1 above)
Comment (giorgiosironi) [reply activity/activity_InvalidateCdn.py]: Good catch, I think the whole `try` block got removed in a single commit

Comment (giorgiosironi) [reply activity/activity_InvalidateCdn.py]: It also looks like the job of a context manager, if I get time I'll do a follow up PR

**Pull request merged**